test/e2e/step-definitions/journal-steps.ts

```diff
@@ -150,3 +176,25 @@ const closeCandidateDetailsDialog = () => {
   const closeCandidateDetailDialog = element(by.id('closeCandidateDetails'));
   clickElement(closeCandidateDetailDialog);
 };
+
+const startingEarlyTest = (candidateName) => {
+  const buttonElement = getElement(by.xpath(`//button/span/h3[text()[normalize-space(.) = "Start test"]]
+    [ancestor::ion-row/ion-col/ion-grid/ion-row/ion-col/candidate-link/div/button/span/
+    h3[text() = "${candidateName}"]]`));
+  clickElement(buttonElement);
+
+  const testEarlyDialog = getElement(by.xpath(`/html/body/ion-app/ion-modal/div/
+  journal-early-start-modal/ion-card/modal-alert-title/ion-row[2]/ion-col/h2`));
+  return expect(testEarlyDialog.isPresent()).to.eventually.be.true;
+};
+
+const startingExpiredTest = (candidateName) => {
+  const buttonElement = getElement(by.xpath(`//button/span/h3[text()[normalize-space(.) = "Start test"]]
+    [ancestor::ion-row/ion-col/ion-grid/ion-row/ion-col/candidate-link/div/button/span/
+    h3[text() = "${candidateName}"]]`));
+  clickElement(buttonElement);
+
+  const testExpireDialog = getElement(by.xpath(`/html/body/ion-app/ion-modal/div/journal-rekey-modal/ion-card/
+    modal-alert-title/ion-row[2]/ion-col/h2`));
+  return expect(testExpireDialog.isPresent()).to.eventually.be.true;
```

Comment:
can you create a helper function for the expect assertion

Reply:
Done.